### ISSUE TITLE
Question / Documentation: How to use the `Uuid` type with postgres (ToSql trait)?

### ISSUE THREAD

**Issue by jrmiller82** [SergioBenitez/Rocket#993]
No matter what I do, I can't seem to get the rocket_contrib Uuid or the uuid crate Uuid to play nice with postgres. I keep getting: 

```
 the trait `rocket_contrib::databases::postgres::types::ToSql` is not implemented for `rocket_contrib::uuid::Uuid`
```
Anyone have any ideas? Thanks.

Edit: Another question: How do I turn on specific postgres features such as `with-uuid` and `with-chrono` per https://github.com/sfackler/rust-postgres#optional-features . I'd like to have uuid and chrono working.

**Comment by jrmiller82** [comment]
So, I figured it out; I needed to load postgres as a dependency and load its features there. Maybe the docs should be updated to reflect that simply loading the rocket_contrib feature may not be enough for the given database?

**Comment by jebrosen** [comment]
Do you think something like the following text, perhaps in https://rocket.rs/v0.4/guide/state/#databases, would be sufficient?

> If your application depends on non-default features of a database engine
> such as support for `chrono` or `uuid`, you will need to enable those features yourself
> by adding them to `Cargo.toml`:
>
> ```toml
> [dependencies]
> postgres = { version = "0.15", features = ["with-chrono"] }
> ```

This also means we should document the correct versions of database crates that `rocket_contrib` interacts with, which is something I think we should start doing anyway.

**Comment by jrmiller82** [comment]
Yes, I think that would definitely help and would have saved me some time today. :)

**Comment by jebrosen** [comment]
#1017 was merged into master and also backported to 0.4.2.
